caluma_interval/tests/test_cli.py

```diff
@@ -21,6 +21,37 @@ def test_main(sys_argv_handler, create_form_to_workflow, cleanup_db):
     main()
 
 
+def test_main_faulty_interval(
+    client, sys_argv_handler, create_form_to_workflow, cleanup_db
+):
+    """
+    Test handling of faulty intervals.
+
+    We just want to know if this runs through without throwing any exception.
+
+    This should never be needed, as Calumas ValidationClass should already
+    enforce this.
+
+    Nonetheless, here's the test:
+    """
+    query = """\
+mutation MyForm {
```

Comment:
I'd indent that for readability (I know we didn't do this before, but some whitespace shouldn't hurt